x-pack/plugins/security_solution/public/flyout/document_details/right/header.tsx

```diff
@@ -38,16 +44,48 @@ export const PanelHeader: FC<PanelHeaderProps> = memo(
     const { dataFormattedForFieldBrowser } = useRightPanelContext();
     const { isAlert } = useBasicDataFromDetailsData(dataFormattedForFieldBrowser);
     const onSelectedTabChanged = (id: RightPanelPaths) => setSelectedTabId(id);
-    const renderTabs = tabs.map((tab, index) => (
-      <EuiTab
-        onClick={() => onSelectedTabChanged(tab.id)}
-        isSelected={tab.id === selectedTabId}
-        key={index}
-        data-test-subj={tab['data-test-subj']}
-      >
-        {tab.name}
-      </EuiTab>
-    ));
+
+    const tourAnchor = useMemo(
+      () =>
+        isAlert
+          ? {
+              'tour-step': getTourAnchor(
+                AlertsCasesTourSteps.reviewAlertDetailsFlyout,
+                SecurityStepId.alertsCases
+              ),
+            }
+          : {},
+      [isAlert]
+    );
+
+    const renderTabs = tabs.map((tab, index) =>
+      isAlert && tab.id === 'overview' ? (
```

Comment:
Same reason as this one: https://github.com/elastic/kibana/pull/181442/files#r1586866840